.github/workflows/containers.yaml

```diff
@@ -53,14 +54,15 @@ jobs:
         uses: redhat-actions/buildah-build@v2
         with:
           image: freeipa-openshift-container
-          tags: ${{ steps.getTags.outputs.TAGS }}
+          tags: ${{ steps.getParams.outputs.TAGS }}
           dockerfiles: |
             ./Dockerfile
           build-args: |
-            PARENT_IMG=${{ secrets.PARENT_IMG }}
-            QUAY_EXPIRATION=${{ steps.getTags.outputs.QUAY_EXPIRATION }}
+            PARENT_IMG=${{ steps.getParams.outputs.PARENT_IMG }}
+            QUAY_EXPIRATION=${{ steps.getParams.outputs.QUAY_EXPIRATION }}
 
       - name: Push To quay.io
+        if: ${{ github.event_name != 'pull_request' }}
```

Comment:
``if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true``

Reply:
Secrets are not accessible on pull_request event. I will restore the initial filter. Thanks for the suggestion.